data/data/agent/files/usr/local/bin/agent-gather

```diff
@@ -10,6 +10,10 @@ function gather_agent_data() {
 	( >&2 echo -n "Gathering agent installation data" )
 	mkdir -p "${ARTIFACTS_DIR}/etc"
 	cp -a /etc/assisted{,-service} "${ARTIFACTS_DIR}/etc/"
+        # Redact pull secrets
+        for manifest in "${ARTIFACTS_DIR}"/etc/assisted/manifests/*; do
+            sed -i -e '/"auth":/ s/: *"[A-Za-z0-9+/]*=*"/: "<redacted>"/g' "${manifest}"
+        done
```

Comment:
What about the journals? They could also potentially capture confidential data (secrets included)

Reply:
They shouldn't, but it's hard to know how to stop them if they did.

Reply:
OK, I figured out a way to do it.